packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -123,9 +141,7 @@ export default function ShadowsEditPanel() {
 	};
 
 	const handleShadowDelete = () => {
-		const updatedShadows = shadows.filter( ( s ) => s.slug !== slug );
-		setShadows( updatedShadows );
-		goTo( `/shadows` );
+		setShadows( shadows.filter( ( s ) => s.slug !== slug ) );
```

Comment:
No need to call `goTo` anymore here, since deleting the shadow will cause the `useEffect` added above to run, and `currentShadow` will be `undefined`